lib/features/modeling/behavior/LabelBehavior.js

```diff
@@ -47,7 +47,7 @@ import {
 import {
   getDistancePointLine,
   perpendicularFoot
-} from './util/GeometricUtil';
+} from 'diagram-js/lib/layout/GeometricUtil';
```

Comment:
Do we consider this a breaking change? 

If so we could re-export this in `modeling/behavor/util/GeometricUtil` to not make it breaking.

Reply:
I was considering it, but we don't use it outside of bpmn-js, so I just moved it.

Of course we can re-export so it is not breaking in case someone else does depend on it

Reply:
> Of course we can re-export so it is not breaking in case someone else does depend on it

So your suggestion is to not treat it as breaking (but an internal)? OK for me.

Reply:
No, let's re-export it. Its easy for us and might save someone pain when updating :slightly_smiling_face:

Reply:
cf. [5c8730d](https://github.com/bpmn-io/bpmn-js/pull/1661/commits/5c8730df3353e087af3b78daad365527c8dd4212)